### DISCUSSION TITLE
希望R68S的下一版固件添加 “IP/MAC绑定”功能

### DISCUSSION THREAD

**Discussion by goodboyspy** [coolsnowwolf/lede#9580] [R68S / 电犀牛专区]
具体来说就是希望增加 luci-app-arpbind和luci-i18n-arpbind-zh-cn这两个软件包和与之配套的关联ipk包。

**Comment by kino516** [comment]
在DHCP/DNS设置内有静态地址分配功能就是你说的IP/MAC绑定。我绑定IP就是在这里设置的。

**Reply by goodboyspy** [reply]
那个是DHCP静态分配，和这个ip/mac绑定是互补的关系，其实这个 luci-app-arpbind 应该叫做ARP绑定更准确
